examples/replay/config-0.25-degree.yaml

```
@@ -211,3 +211,79 @@ FV3Dataset:
         - tmp
         - ugrd
         - vgrd
+
+MOM6Dataset:
+    file_prefixes:
+        - ocn_
+    
+    path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/03h-freq/zarr/"
```

I just realized we should also change this outer directory to reflect the 06h frequency
```suggestion
    path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/06h-freq/zarr/"
```